mixpanel/mixpanel.d.ts

```diff
@@ -26,7 +26,7 @@ interface Mixpanel
 
     unregister(propertyName:string):void;
 
-    identify(id:string):void;
+    identify(id?:string):void;
```

Comment:
@vvakame This is what one of the Engineers at Mixpanel told me to do:

> If you pass no arguments to mixpanel.identify() it will use the distinct_ID that's already in the cookie.
> Aaron Krivitzky | Core Sales Engineer

Personally, I don't think they have great documentation.